package.json

```
@@ -17,12 +20,25 @@
     "react-spinners": "^0.6.1",
     "styled-components": "^4.4.1"
   },
+  "author": {
+    "email": "anongharbia@gmail.com",
+    "name": "Awni Zaher Gharbia",
+    "url": "https://awnigharbia.github.io/filmy/#/"
```

```suggestion
    "url": "https://awnigharbia.netlify.app/"
```

This your personal website. You don't put the filmy URL there. You already have `homepage` set to filmy URL.

---

Oh yeah!